curated_transformers/tests/models/gpt_neox/test_decoder.py

```diff
@@ -32,5 +35,22 @@ def test_decoder_with_torch_compile(torch_device):
         torch_device,
         with_cache=False,
         with_positions=False,
-        with_torch_compile=True,
+        jit_method=JITMethod.TorchCompile,
+        with_torch_sdp=with_torch_sdp,
+    )
+
+
+@pytest.mark.slow
+@pytest.mark.skipif(not has_hf_transformers, reason="requires huggingface transformers")
+@pytest.mark.parametrize("torch_device", TORCH_DEVICES)
+@pytest.mark.parametrize("with_torch_sdp", [False, True])
+def test_decoder_with_torchscript_trace(torch_device, with_torch_sdp):
+    assert_decoder_output_equals_hf(
+        GPTNeoXDecoder,
+        "trl-internal-testing/tiny-random-GPTNeoXForCausalLM",
+        torch_device,
+        # with_cache=False,
+        # with_positions=False,
```

Comment:
```suggestion
        with_cache=True,
        with_positions=True,
```
To make it more explicit.